modules/@apostrophecms/piece-type/index.js

```diff
@@ -100,48 +100,60 @@ module.exports = {
     add: {
       archive: {
         label: 'apostrophe:archive',
+        route: '/archive',
+        // TEMP - full batch operation work is upcoming
+        messages: {
+          progress: 'Archiving {{ type }}...',
+          completed: 'Archived {{ count }} {{ type }}.'
+        },
         icon: 'archive-arrow-down-icon',
         if: {
           archived: false
         }
       },
       restore: {
         label: 'apostrophe:restore',
+        route: '/restore',
+        // TEMP - full batch operation work is upcoming
+        messages: {
+          progress: 'Restoring {{ type }}...',
+          completed: 'Restoring {{ count }} {{ type }}.'
+        },
         icon: 'archive-arrow-up-icon',
         if: {
           archived: true
         }
       }
+      // visibility: {
+      //   label: 'apostrophe:visibility',
+      //   requiredField: 'visibility',
+      //   fields: {
+      //     add: {
+      //       visibility: {
+      //         type: 'select',
+      //         label: 'apostrophe:visibilityLabel',
```

Comment:
If  this isn't present the i18n linter warns that `visibilityLabel` isn't used. We might bring this back before the feature release, so I'll leave it in for now.